### PR TITLE
docs(refactor): review docs MTA-1808

### PR DESCRIPTION
**Pull request by bene2k1**

### CODE REVIEW

File: dedibox-network/ip-failover/how-to/configure-freebsd.mdx [hunk 1]
```diff
@@ -6,15 +6,15 @@ content:
   h1: How to configure a failover IP on FreeBSD
   paragraph: This page explains how to configure a failover IP on Debian or FreeBSD.
 dates:
-  validation: 2021-07-29
+  validation: 2022-02-28
   posted: 2021-07-29
 ---
 
-You will find below, information on how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on FreeBSD.
+Below you find information on how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on FreeBSD.
```
Comment (ldecarvalho-doc) [inline dedibox-network/ip-failover/how-to/configure-freebsd.mdx:13]:
```suggestion
This page shows you how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on FreeBSD.
```

File: dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx [hunk 2]
```diff
@@ -6,22 +6,22 @@ content:
...
 dates:
-  validation: 2021-07-29
+  validation: 2022-02-28
   posted: 2021-07-29
 ---
 
 You can use the same virtual MAC at several failover IPs pointing on the same dedicated server.
 
-To do this you need to have several [failover IPs](/dedibox-network/ip-failover/concepts/#failover-ips) at the same server and at least one [virtual MAC](/dedibox-network/ip-failover/concepts/#virtual-mac-address) associated with one of these failover IPs.
+To do this, you need to have several [failover IPs](/dedibox-network/ip-failover/concepts/#failover-ips) at the same server and at least one [virtual MAC](/dedibox-network/ip-failover/concepts/#virtual-mac-address) associated with one of these failover IPs.
```
Comment (ldecarvalho-doc) [inline dedibox-network/ip-failover/how-to/configure-multiip-virtualmac.mdx:15]: maybe this should go in the requirements?

File: dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx [hunk 3]
```diff
@@ -6,18 +6,19 @@ content:
   h1: How to the reverse DNS of a failover IP
   paragraph: This page explains how to configure the reverse DNS for a failover IP.
 dates:
-  validation: 2021-07-29
+  validation: 2022-02-28
   posted: 2021-07-29
 ---
 
-Failover IP addresses allow to have several IPs on the server and thus, for example, to have one IP for each website or for each virtual machine.
+Failover IP addresses allow several IPs on the server and thus, for example, one IP for each website or virtual machine.
```
Comment (ldecarvalho-doc) [inline dedibox-network/ip-failover/how-to/configure-reverse-dns.mdx:13]:
```suggestion
Failover IP addresses allow you to have several IP addresses with your server. This allows you to assign IP addresses to different websites or to virtual machines.
```

File: dedibox-network/ip-failover/how-to/configure-windows.mdx [hunk 4]
```diff
@@ -6,20 +6,20 @@ content:
   h1: How to configure a failover IP on Windows Server 2019
   paragraph: This page explains how to configure a failover on Windows Server 2019.
 dates:
-  validation: 2021-07-29
+  validation: 2022-02-28
   posted: 2021-07-29
 ---
 
-You will find below, information on how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on Windows Server 2019.
+Below you find information on how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on Windows Server 2019.
```
Comment (ldecarvalho-doc) [inline dedibox-network/ip-failover/how-to/configure-windows.mdx:13]:
```suggestion
This page shows you how to configure a [failover IP](/dedibox-network/ip-failover/concepts/#failover-ips) on Dedibox servers running on Windows Server 2019.
```

**Review by RoRoJ — APPROVED** [review]

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**